adobe_vipm/management/commands/sync_3yc_enrollments.py

```diff
@@ -1,28 +1,26 @@
-from django.core.management.base import BaseCommand
 from mpt_extension_sdk.core.utils import setup_client
 
 from adobe_vipm.flows.sync import sync_agreements_by_3yc_enroll_status
+from adobe_vipm.management.commands.base import AdobeBaseCommand
 
 
-class Command(BaseCommand):
+class Command(AdobeBaseCommand):
```

Comment:
@jentyk  @alephsur  what is the difference between this command and process_3yc?